### PR TITLE
25 lib rs boilerplate

### PR DESCRIPTION
**Pull request by Connoropolous**
we merged this into the wrong branch in #72 !

@sphinxc0re is going to follow up with a macro that fills in this file

### CODE REVIEW

File: src/cli/scaffold/rust/Cargo.template.toml [hunk 1]
```diff
@@ -7,8 +7,9 @@
   serde = "1.0"
   serde_json = "1.0"
   serde_derive = "1.0"
-  hdk = { git = "https://github.com/holochain/holochain-rust" , branch = "develop" }
-  holochain_wasm_utils = { git = "https://github.com/holochain/holochain-rust" , branch = "develop" }
+  boolinator = "2.4"
```
Comment (sphinxc0re) [inline src/cli/scaffold/rust/Cargo.template.toml]: I don't think that boolinator should be included in the scaffold. It is pretty confusing

**Review by sphinxc0re — CHANGES_REQUESTED** [review]

File: src/cli/scaffold/rust/lib.rs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#[macro_use]
+extern crate hdk;
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
+#[macro_use]
+extern crate serde_json;
+extern crate boolinator;
+
+// see https://holochain.github.io/rust-api/0.0.1/hdk/ for info on using the hdk library
```
Comment (sphinxc0re) [inline src/cli/scaffold/rust/lib.rs]: Should this file stay this way? Or should we put in a basic zome and entry definition?

**Review by sphinxc0re — CHANGES_REQUESTED** [review]

File: src/cli/scaffold/rust/lib.rs (hunk 2 above)
Comment (Connoropolous) [reply src/cli/scaffold/rust/lib.rs]: I'm not sure... what do you think?

File: src/cli/scaffold/rust/Cargo.template.toml (hunk 1 above)
Comment (Connoropolous) [reply src/cli/scaffold/rust/Cargo.template.toml]: Ok, sure yes let's remove

File: src/cli/scaffold/rust/lib.rs (hunk 2 above)
Comment (sphinxc0re) [reply src/cli/scaffold/rust/lib.rs]: I think we have to put in some basic zome definition

**Review by sphinxc0re — APPROVED** [review]

File: src/cli/scaffold/rust/lib.rs (hunk 2 above)
Comment (Connoropolous) [reply src/cli/scaffold/rust/lib.rs]: I agree, that'd be good, and also I think be required to get tests passing

**Review by zippy — APPROVED** [review]

**Pull request merged**